.travis.yml

```diff
@@ -1,12 +1,12 @@
-language: objective-c
```

Comment:
@hetadhruv @yao-lu-1216 I think this file needs to be removed and not commented.